modules/apps/portal-search/portal-search/src/main/java/com/liferay/portal/search/internal/IndexerRegistryImpl.java

```diff
@@ -136,13 +138,17 @@ protected void activate(
 
 		_bundleContext = bundleContext;
 
-		modified(properties);
-
 		_indexerPostProcessorsServiceTrackerMap =
 			ServiceTrackerMapFactory.openMultiValueMap(
 				_bundleContext, IndexerPostProcessor.class,
 				"indexer.class.name");
 
+		_indexerRequestBufferOverflowHandler =
```

Comment:
Maybe we should initialize this variable into the modified method. What do you think?